### ISSUE TITLE
Looking for 2000's style emoticon set for Discord Messenger

### ISSUE THREAD

**Issue by iProgramMC** [DiscordMessenger/dm#81]
Options considered so far:

* Famfamfam's silk icon set: Very few emoticons

* MSN Messenger: Risky, could cause copyright issues, ~47 emoticons for MSN 4 and 5, MSN 6 and up has blended white background into the emoticons making them unusable

* Apple's emoticon set: Risky, could cause copyright issues, hard to extract from software

**Comment by zaafonin** [comment]
There's this one which looks really 2000s (a lot of pixel-perfect icons similar to 2000s web design): https://vk.com/images/emoji/sprites/sprite0.png, sprite1.png, sprite2.png
![image](https://github.com/DiscordMessenger/dm/assets/21102341/7d24f5b0-2f13-4af4-b665-641de25b6aa0)

**Comment by iProgramMC** [comment]
Looks good :)

**Comment by zaafonin** [comment]
Yeah here they are. Looks like a lot of them are rescales of Apple's emojis, but a lot of them are unique.
![sprite_2](https://github.com/DiscordMessenger/dm/assets/21102341/e199c0e9-b5ed-4163-945f-93de672d8762)
![sprite_1](https://github.com/DiscordMessenger/dm/assets/21102341/802beb6b-335e-4f93-a24a-1c1215124fe6)
![sprite_0](https://github.com/DiscordMessenger/dm/assets/21102341/de45c173-8f10-41e3-9d25-a7634577588f)
![sprite_9](https://github.com/DiscordMessenger/dm/assets/21102341/a90134de-b7cf-4320-abbf-92d57839f485)
![sprite_8](https://github.com/DiscordMessenger/dm/assets/21102341/39e62a36-e4d3-49ba-93fe-a1704a135b66)
![sprite_7](https://github.com/DiscordMessenger/dm/assets/21102341/4126ed62-e11e-471e-a870-d1561417aebd)
![sprite_6](https://github.com/DiscordMessenger/dm/assets/21102341/0fe7850e-d419-4bbe-90f0-2dbf08d3bff7)
![sprite_5](https://github.com/DiscordMessenger/dm/assets/21102341/d94c9313-da50-4a02-8076-4493da7e5c4b)
![sprite_4](https://github.com/DiscordMessenger/dm/assets/21102341/2cbdfce2-d4c1-4b34-bd3c-da5d9e46a48e)
![sprite_3](https://github.com/DiscordMessenger/dm/assets/21102341/6d9623db-6391-40d0-850e-ff9c6c28a539)

**Comment by zaafonin** [comment]
Quite the opposite. They drew their own emojis when there weren't as much of them. For country flags, looks like at first they've filled post-Soviet countries + most "known" outside of that

**Comment by iProgramMC** [comment]
I see. Perhaps I could find earlier versions of these emoji sheets to use. Where did you get the other ones? (urls) I tried sprite_3.png from the format you gave me but it gave me a blank page.

**Comment by zaafonin** [comment]
From what I see they changed how sprite sheets are fetched when more emojis were needed. 0, 1 and 2 contain no Apple emojis, but starting from 3 there are some. I suspect they still draw emojis that don't have as many variations, but they've given up on skin tones

**Comment by rari-teh** [comment]
It would probably be way too much work to bring any of these sets up to date, but another option could be Facebook’s original emoji and emoticon set. It has only something like 300-ish glyphs, though. [This website](http://www.emoticonplanet.com/images/facebook-emoticons/) has the full original set. Other possibilities include [HTC’s set](https://emojipedia.org/htc/sense-8) from 2016, [SoftBank’s latest set](https://emojipedia.org/softbank) from 2014 and [au by KDDI Type D-3](https://emojipedia.org/au-kddi/type-d-3) (2004–2012).

![Sample](https://github.com/DiscordMessenger/dm/assets/32584598/18eee471-33aa-4945-9ad9-96e94688010c)

**Comment by 0penrc** [comment]
stellar from psi
![image](https://github.com/DiscordMessenger/dm/assets/80153347/2475824d-979d-4990-9567-f86a7208985b)

**Comment by 0penrc** [comment]
https://commons.wikimedia.org/wiki/Tango_icons#Emotes

tango emoticons

**Comment by 0penrc** [comment]
more tango emotes
https://commons.wikimedia.org/wiki/GNOME_Desktop_icons#Emotes

**Comment by 0penrc** [comment]
crystal emoticons

https://www.cinnamon-look.org/s/Cinnamon/p/1120232

**Comment by 0penrc** [comment]
cutemoticons: https://www.pling.com/p/1108677/
icq (probably non-free?): https://www.pling.com/p/1107337/
donanimhaber forum: https://www.pling.com/p/1002209/
twosouls (modified msn, probably non-free) https://www.pling.com/p/1002261/
gadu gadu: https://www.pling.com/p/1002207/
yahoo (probably non free): https://www.pling.com/p/1002264/
meebo (probably non-free): https://www.pling.com/p/1011923/
kde forum: https://www.pling.com/p/1002278/
**proticon (this is one of my favorites): https://www.pling.com/p/1012270/**
qip (probably non-free): https://www.pling.com/p/1002257/
ipb: https://www.pling.com/p/1108784/
valknut: https://www.pling.com/p/1002236/
cartoon: https://www.pling.com/p/1120523/
amsn (might be non-free? im not sure): https://www.pling.com/p/1112351/
aim (probably non-free): https://www.pling.com/p/1002196/
simple: https://www.pling.com/p/1002213/
kolobok: https://www.pling.com/p/1002303/
qip (probably non-free): https://www.pling.com/p/1002262/
emotikons: https://www.pling.com/p/1120525/
peerweb: https://www.pling.com/p/1002211/
crystal msn: https://www.pling.com/p/1119387/
plainicons: https://www.pling.com/p/1012051/

**Comment by zaafonin** [comment]
ICQ = Kolobok = QIP. The author even has a kit for creating your own smileys. Its license requires attribution and prohibits commercial use. The whole set is a gem of 2000s internet forums (especially Russian-speaking), but these smileys aren't really related to Unicode emojis (which makes them really unique but hard to fit into modern context) https://www.kolobok.us/download.php?view.67

**Comment by rari-teh** [comment]
These remind me of deviantART’s old emoticons. Same thing, cool and interesting and a little window into 2000s forum culture, but very much in the “emoticons, not emoji” park lol. FWIW they can be found on [the Wayback Machine](http://web.archive.org/web/*/https://e.deviantart.net/emoticons/*).

**Comment by 0x5066** [comment]
What about the emojis from the SerenityOS project?

**Comment by 0x5066** [comment]
Oh, whoops 😅

**Comment by RF7391** [comment]
My solution to this issue:
Make it use emoji packs so that we can make custom emojis, it'll help with copyright issues imo

**Comment by iProgramMC** [comment]
I already thought of that. See the initial comment. But I am looking for a *default* pack.

**Comment by freedom7341** [comment]
https://www.phpbb.com/customise/db/styles/smilies-13

Could be some packs of interest here.

**Comment by superandrews13** [comment]
stumbled across this wip emoji pack that seems to check most, if not all of the boxes https://github.com/vercte/forumoji

**Comment by channel77** [comment]
When will this be implemented, along with Discord's emoji chooser pop-up?

**Comment by iProgramMC** [comment]
I know this isn't "2000s" style but https://github.com/joypixels/emoji-toolkit - the old versions are relatively free. Just need to give attribution.

**Comment by wertzuz** [comment]
https://gh.vercte.net/forumoji/

**Comment by imadeagithubaccountforthis** [comment]
actually does anyone have a pack of the transparent MSN Messenger emojis

**Comment by wertzuz** [comment]
Not exactly a pack, but should be ok: https://www.messentools.com/en/msn-emoticons/msn/
(don't think they are transparent unfortunatly)
> actually does anyone have a pack of the transparent MSN Messenger emojis

**Comment by DarazsBalintArp** [comment]
on it! (custom)

<img width="906" height="435" alt="Image" src="https://github.com/user-attachments/assets/943dd77b-57b1-4538-b2a2-99ae2ccac40a" />

**Comment by iProgramMC** [comment]
These look cool but I'm unlikely to integrate them into DM. I'm looking for a reasonably complete set (so at least supporting Emoji 1.0)

**Comment by DarazsBalintArp** [comment]
Obviously its not fully done yet. I just thought why not start make a custom one and then hear feedback if it would match the requirement, then i would continue it.

**Comment by vercte** [comment]
> > https://gh.vercte.net/forumoji/
> 
> Thanks! This looks pretty much perfect!
> 
> I will start implementing emoji support at some point in the future using forumoji. (Unless anyone has anything better)

I just randomly found this thread when looking for one of my repos. If you need help implementing, feel free to ask me :)

**Comment by a-crazy-pencil** [comment]
@iProgramMC what about https://r74n.com/moji/
it is free it looks like 2000s emjois and you can ask R74n if they can make more

**Comment by Preloading** [comment]
not truely complete, but the emoticons from OpenLiveWriter, formerly Windows Live Writer, which appear to be the ones from Windows Live Messenger 201*, but it's under an MIT license so it's fine to yoink. https://github.com/OpenLiveWriter/OpenLiveWriter/tree/f86306f8d69c4b487cb14a88a08bf1d7669bc9cf/src/managed/OpenLiveWriter.Localization/Emoticons. Also doesn't have the white background issues :)
